go.mod

```diff
@@ -43,4 +43,4 @@ require (
 	gopkg.in/check.v1 v1.0.0-20200902074654-038fdea0a05b // indirect
 )
 
-go 1.22.2
+go 1.23.3
```

Comment:
do you wanna do this in this PR?

Reply:
might as well

Reply:
oh wait, this is from vitess having a newer version of go...